### PR TITLE
Add German translation to AppData

### PR DESCRIPTION
**Pull request by JakobDev**
This PR just adds a German translation to the AppData file.

For the future it might be better to translate the AppData file with the translation files, using something like [ITS Tool](http://itstool.org/index.html), so it can be translated using Weblate.

### CODE REVIEW

**Comment by SmallJoker** [comment]
Where is this information exposed (on Android)? Who will benefit from it? If nobody ever gets to see this data, it's nonsense to spend efforts into maintaining/translating it.

**Comment by JakobDev** [comment]
The AppData file is used by Linux Software centres. It's get seen by every Linux User who use a graphic Program to install Minetest.

File: misc/net.minetest.minetest.appdata.xml [hunk 1]
```diff
@@ -11,26 +11,44 @@
...
 			Minetest is designed to be simple, stable, and portable.
 			It is lightweight enough to run on fairly old hardware.
+		</p><p xml:lang="de">
+			Minetest ist designt um einfach stabil und portabel zu sein.
+			Es ist leichtgewichtig genug, um auf relativ alter Hardware zu laufen.
 		</p><p>
 			Minetest has many features, including:
+		</p><p xml:lang="de">
+			Mintest besitzt viele Features, unteranderem:
 		</p>
```
Comment (sfan5) [inline misc/net.minetest.minetest.appdata.xml:38]: Shouldn't they be grouped more like this?
```
<p>English description</p>
<p>sample
text
abc</p>
<p xml:lang="de">Deutsche Beschreibung</p>
<p xml:lang="de">beispiel
text
abc</p>
```

**Comment by Wuzzy2** [comment]
> Minetest ist ein Sandkastenspiel und Spielengine mit unendlichen Welten

Missing period at the end. “und Spielengine” → “und eine Spielengine”.

> Spieler können in einer offenen 3D Welt viele verschiedene Arten von
> Blöcken platzieren und abbauen.

 “3D Welt” → “3-D-Welt” or “3D-Welt”.

https://deppenleerzeichen.de/

:P

> Mintest besitzt viele Features, unteranderem:

“Mintest” → “Minetest”. “unteranderem” → “unter anderem”.

> Minetest ist designt um einfach stabil und portabel zu sein.

Missing comma after “designt”.

**Comment by Wuzzy2** [comment]
@sfan5: IIRC the spec says the translated text element must follow directly after the English one. But for some reason I can't find the spec anymore. :-( Anyway, if my memory is right, the current structure in this PR is correct then.

**Comment by Wuzzy2** [comment]
> Items können aus Rohmaterialien gecraftet werden

“Dinge können aus Rohmaterialien gefertigt werden” (Kein Denglisch, bitte. / No Denglisch [German-English perversion], please.)

**Comment by JakobDev** [comment]
I had applied the correction from @Wuzzy2 .

> But for some reason I can't find the spec anymore. 

The Spec can be found [here](https://www.freedesktop.org/software/appstream/docs/), but it doesn't say anything about the structure of translations. The structure in this PR is the correct one, since is used by all other AppStream files.

File: misc/net.minetest.minetest.appdata.xml [hunk 2]
```diff
@@ -16,31 +16,31 @@
 		<p>
 			Minetest is an infinite-world block sandbox game and game engine.
 		</p><p xml:lang="de">
-			Minetest ist ein Sandkastenspiel und Spielengine mit unendlichen Welten
+			Minetest ist ein Sandkastenspiel und eine Spielengine mit unendlichen Welten.
```
Comment (SwissalpS) [inline misc/net.minetest.minetest.appdata.xml:19]: Is there really no German word to express game engine? I'm not 100%, but wouldn't it be "Spiel-Engine" if Deutschified?

File: misc/net.minetest.minetest.appdata.xml [hunk 3]
```diff
@@ -16,31 +16,31 @@
...
-			Spieler können in einer offenen 3D Welt viele verschiedene Arten von
+			Spieler können in einer offenen 3D-Welt viele verschiedene Arten von
 			Blöcken platzieren und abbauen. Dies erlaubt das Bauen von allen möglichen
 			Strukturen im Einzelspieler oder auf Mehrspielerservern.
 		</p><p>
 			Minetest is designed to be simple, stable, and portable.
 			It is lightweight enough to run on fairly old hardware.
 		</p><p xml:lang="de">
-			Minetest ist designt um einfach stabil und portabel zu sein.
+			Minetest ist designt, um einfach stabil und portabel zu sein.
```
Comment (SwissalpS) [inline misc/net.minetest.minetest.appdata.xml:32]: designt -> entworfen

File: misc/net.minetest.minetest.appdata.xml [hunk 4]
```diff
@@ -16,31 +16,31 @@
...
+			Minetest ist ein Sandkastenspiel und eine Spielengine mit unendlichen Welten.
 		</p><p>
 			Players can create and destroy various types of blocks in a
 			three-dimensional open world. This allows forming structures in
 			every possible creation, on multiplayer servers or in singleplayer.
 		</p><p xml:lang="de">
-			Spieler können in einer offenen 3D Welt viele verschiedene Arten von
+			Spieler können in einer offenen 3D-Welt viele verschiedene Arten von
 			Blöcken platzieren und abbauen. Dies erlaubt das Bauen von allen möglichen
 			Strukturen im Einzelspieler oder auf Mehrspielerservern.
```
Comment (SwissalpS) [inline misc/net.minetest.minetest.appdata.xml:27]: again I'm not 100%, ... im Einzelspieler- oder auf Mehrspieler-Servern.

Comment (SwissalpS) [reply misc/net.minetest.minetest.appdata.xml:27]: this might be a CH vs DE thing.

File: misc/net.minetest.minetest.appdata.xml [hunk 5]
```diff
@@ -11,26 +11,44 @@
...
 			Minetest is an infinite-world block sandbox game and game engine.
+		</p><p xml:lang="de">
+			Minetest ist ein Sandkastenspiel und eine Spielengine mit unendlichen Welten.
 		</p><p>
 			Players can create and destroy various types of blocks in a
 			three-dimensional open world. This allows forming structures in
 			every possible creation, on multiplayer servers or in singleplayer.
+		</p><p xml:lang="de">
+			Spieler können in einer offenen 3D-Welt viele verschiedene Arten von
+			Blöcken platzieren und abbauen. Dies erlaubt das Bauen von allen möglichen
```
Comment (appgurueu) [inline misc/net.minetest.minetest.appdata.xml:26]: > Dies erlaubt das Bauen von allen möglichen Strukturen

Should probably be worded differently (das Bauen von allerlei/verschiedensten/vielfältigen/... Strukturen), although I fail to grasp the intention of the English version ("in every possible creation").

File: misc/net.minetest.minetest.appdata.xml [hunk 6]
```diff
@@ -11,26 +11,44 @@
...
+			Strukturen im Einzelspieler oder auf Mehrspielerservern.
 		</p><p>
 			Minetest is designed to be simple, stable, and portable.
 			It is lightweight enough to run on fairly old hardware.
+		</p><p xml:lang="de">
+			Minetest ist designt, um einfach stabil und portabel zu sein.
```
Comment (appgurueu) [inline misc/net.minetest.minetest.appdata.xml:32]: "einfach, stabil und portabel" (Aufzählung) statt "einfach stabil und portabel"

File: misc/net.minetest.minetest.appdata.xml [hunk 7]
```diff
@@ -11,26 +11,44 @@
...
+			Es ist leichtgewichtig genug, um auf relativ alter Hardware zu laufen.
 		</p><p>
 			Minetest has many features, including:
+		</p><p xml:lang="de">
+			Minetest besitzt viele Features, unter anderem:
 		</p>
 		<ul>
 			<li>Ability to walk around, dig, and build in a near-infinite voxel world</li>
+			<li xml:lang="de">Die Möglichkeit, in einer nahezu unendlichen Voxel-Welt herumzulaufen, zu graben und zu bauen</li>
```
Comment (appgurueu) [inline misc/net.minetest.minetest.appdata.xml:41]: Maybe use "abbauen" statt "graben"

File: misc/net.minetest.minetest.appdata.xml [hunk 8]
```diff
@@ -11,26 +11,44 @@
...
 			<li>Crafting of items from raw materials</li>
+			<li xml:lang="de">Items können aus Rohmaterialien gefertigt werden</li>
 			<li>Fast and able to run on old and slow hardware</li>
+			<li xml:lang="de">Gute Performance selbst auf älterer und langsamer Hardware>/Li>
```
Comment (appgurueu) [inline misc/net.minetest.minetest.appdata.xml:45]: "Performanz" would work as well

File: misc/net.minetest.minetest.appdata.xml (hunk 2 above)
Comment (sfan5) [reply misc/net.minetest.minetest.appdata.xml:19]: wp has the spelling with a hyphen https://de.wikipedia.org/wiki/Spiel-Engine

File: misc/net.minetest.minetest.appdata.xml (hunk 5 above)
Comment (sfan5) [reply misc/net.minetest.minetest.appdata.xml:26]: I vote for "vielfältigen", it's a great word

File: misc/net.minetest.minetest.appdata.xml (hunk 4 above)
Comment (sfan5) [reply misc/net.minetest.minetest.appdata.xml:27]: alternatively: "im Einzel- oder Mehrspielermodus"

File: misc/net.minetest.minetest.appdata.xml [hunk 9]
```diff
@@ -11,26 +11,44 @@
...
 		</p><p>
 			Minetest has many features, including:
+		</p><p xml:lang="de">
+			Minetest besitzt viele Features, unter anderem:
 		</p>
 		<ul>
 			<li>Ability to walk around, dig, and build in a near-infinite voxel world</li>
+			<li xml:lang="de">Die Möglichkeit, in einer nahezu unendlichen Voxel-Welt herumzulaufen, zu graben und zu bauen</li>
 			<li>Crafting of items from raw materials</li>
+			<li xml:lang="de">Items können aus Rohmaterialien gefertigt werden</li>
```
Comment (sfan5) [inline misc/net.minetest.minetest.appdata.xml:43]:
```suggestion
			<li xml:lang="de">Fertigen von Items aus Rohmaterialien</li>
```

File: misc/net.minetest.minetest.appdata.xml (hunk 8 above)
Comment (sfan5) [inline misc/net.minetest.minetest.appdata.xml:45]: `>/Li>` should be `</li>` here

File: misc/net.minetest.minetest.appdata.xml (hunk 6 above)
Comment (SwissalpS) [reply misc/net.minetest.minetest.appdata.xml:32]: designt -> entworfen

File: misc/net.minetest.minetest.appdata.xml (hunk 7 above)
Comment (SwissalpS) [reply misc/net.minetest.minetest.appdata.xml:41]: what about "ausgraben"?

Comment (appgurueu) [reply misc/net.minetest.minetest.appdata.xml:41]: > what about "ausgraben"?

That's misleading: The English translation would be "dig up/out".

Comment (JakobDev) [reply misc/net.minetest.minetest.appdata.xml:41]: I#m not a fan of abbauen. zu abbauen und zu bauen sounds bad.

File: misc/net.minetest.minetest.appdata.xml (hunk 8 above)
Comment (JakobDev) [reply misc/net.minetest.minetest.appdata.xml:45]: Nice catch

**Comment by JakobDev** [comment]
I have now done the changes requested in the discussion

**Review by sfan5 — APPROVED** [review]

**Comment by Wuzzy2** [comment]
This looks also good to me.

**Comment by Zughy** [comment]
I think the PR should be merged, just, I want to underline how the English description of the file is outdated (it calls Minetest also a game, and it's basically describing MTG). So it should be updated eventually, and all the translations should follow.

**Review by SmallJoker — APPROVED** [review]
LGTM. Akzeptable Übersetzung.

**Pull request merged**